src/components/Recipes/AddToMealplanContainer.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { AddRecipeTitleToMealplanToggle } from '../Button'
+import imagePathPlus from '../../images/inverted-plus.svg'
+import { v4 as uuidv4 } from 'uuid'
+
+import styled from 'styled-components'
+import { saveToLocal, loadFromLocal } from '../../utils/localStorage'
+
+export default function AddToMealplanContainer({ title }) {
+  const [active, setActive] = useState(() => {
+    const mealplanList = loadFromLocal('mealplanList') || {
```

Comment:
Wow! Da hast du mich jetzt ganz schön zum Recherchieren gebracht, weil ich bisher das noch nie gesehen hatte, dass man bei useState auch eine Funktion reingeben kann 😉 

Ich würde die Funktion aber auslagern, ich habe erst gar nicht gesehen, dass Zeile 16 - 21 auch noch in dem State sind.